docs/content/components/overlay/dialog/dialog-appearance.demo.tsx

```diff
@@ -11,7 +11,7 @@ export default (props: DialogProps) => (
         details pre-filled, except for the date.
       </Dialog.Content>
       <Dialog.Actions>
-        <Button variant="secondary">Cancel</Button>
+        <Dialog.Cancel>Cancel</Dialog.Cancel>
```

Comment:
Cancel should than only be used for cancel actions? 

I think its strange to write a Dialog.Cancel and than also a normal Button inside the actions?

Reply:
Also there must not be canceled written down

Reply:
I added the `Cancel` to it does not accidentally get translated ;)

Reply:
> I think its strange to write a Dialog.Cancel and than also a normal Button inside the actions?

@sarahgm suggestions?

Reply:
I'm not sure if I would call it Dialog.Cancel, I think its also ok to have a dedicated cancel button for everything

Reply:
But for everything it not possible, this is a "dedicated" button, because you no longer have to use the callback function so you can access the `close` function. Can also name it close instead? But I wasn't sure if this is confusing with the `closeButton` prop on the `<Dialog>`

Reply:
Just my opinion :P I mean this:
```tsx
<Dialog.Actions>
	<Button variant="close"/>
	<Button variant="primary"/>
</Dialog.Actions>
```
But its ok

Reply:
Yeah, but having the `Button` depend on the `OverlayContext` seems really bad.

Reply:
Any thoughts @aromko @OsamaAbdellateef

Reply:
For me it's fine. Also the fallback with button.cancel.

Reply:
So merge it? :P